src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -178,10 +178,10 @@ protected function setIndentLevel(int $level): void
      */
     protected function indent(): void
     {
-        $multiplier = $this->tabOrSpaceIndentCharacter === ' ' ? 4 : 1;
+        $indentSize = $this->rectorConfigProvider->getIndentSize();
```

Comment:
@TomasVotruba this cause bug https://github.com/rectorphp/rector/issues/7638 which crash on tab https://getrector.org/demo/1d6243b2-ab0c-4002-814c-86f36b465412

Reply:
On `tab` `\t` usage, the `$this->rectorConfigProvider->getIndentSize()` seems return 1 ,while using `$this->tabOrSpaceIndentCharacter` above result 4.